tests/test_camera.py

```diff
@@ -1,13 +1,13 @@
-import os
 import unittest
 from unittest.mock import patch
 
 import cv2
+import numpy as np
+from pathlib import Path
```

Comment:
Oh, `pathlib` again. That's still an ongoing discussion, but I'm all for it :grin:

Reply:
> pathlib

Thanks for the explanation. Which module is reliable (stable) to use (os.path or pathlib or others)?

Reply:
Both are stable and reliable, but there are other pros and cons to them. It's just an internal discussion and we haven't completely decided yet.

Reply:
Let's stick to `os.path` for now to keep it in line with the rest of the project

Reply:
Ok